addons/web_tour/static/src/js/tip.js

```diff
@@ -43,20 +43,38 @@ var Tip = Widget.extend({
             top: "50%",
             left: "50%",
         };
+        this.$scrollEvent = {};
```

Comment:
It was just an object at the variable creation. Otherwise, when I do :
`this.$scrollEvent = this.$scrolableElement.on('scroll', () => {
    this.afterScrollAction();
});`
It's a jquery element which is insert in the variable. I guess, only doing `this.$scrollEvent;` and checking if the value is **undefined** before getting its length should resolve the syntax issue right ?

Reply:
I guess, but why do you need scrollEvent at all if it is equal to scrolableElement?

Reply:
At the start, I think I wanted to avoid a case of multiple scroll event (One new every time I called `attach_to`). But I think it's not useful anymore because the code improved and it doesn't call `attach_to` more than once anymore. (And even if we do, we can simply `off()` then `on()` every time)

I'll remove this variable !